From c065b972ed294769a41936d6b9feb336473af5d1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?G=C3=A1bor=20Boskovits?= <boskovits@gmail.com>
Date: Sat, 4 Nov 2017 15:28:47 +0100
Subject: Fix SOURCE_DATE_EPOCH ignored bug (#2735)

* Fix SOURCE_DATE_EPOCH ignored bug

Fix a bug in networkx/release.py that makes build
non-reproducible.
---
 networkx/release.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/networkx/release.py b/networkx/release.py
index e81fc0c0..6322cf0d 100644
--- a/networkx/release.py
+++ b/networkx/release.py
@@ -135,7 +135,7 @@ def get_revision():
 
 def get_info(dynamic=True):
     # Date information
-    date_info = datetime.datetime.now()
+    date_info = datetime.datetime.utcfromtimestamp(int(os.environ.get('SOURCE_DATE_EPOCH', time.time())))
     date = time.asctime(date_info.timetuple())
 
     revision, version, version_info, vcs_info = None, None, None, None
-- 
2.14.2

